about.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>About - Alex Brown</title>
+    <link rel=StyleSheet href="stylesheets/normalize.css" type="text/css" media=screen>
+    <link rel=StyleSheet href="stylesheets/about-style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+  </head>
+
+  <body class="background">
+
+    <nav class="nav-container">
+        <a href="index.html">Home</a>
```

Comment:
Just a suggestion, having a rollover effect on these menu items would make them "pop" a bit more, especially as you do some thing similar in the main page.